.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,61 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+# ******** NOTE ********
+
+name: "CodeQL"
+
+on:
+  pull_request:
+    branches:
+      - master
+      - opendistro-*
+  push:
+    branches:
+      - master
+      - opendistro-*
+  schedule:
+    - cron: '0 */3 * * *'
+
+jobs:
+  analyze: CodeQL-Scan
+    name: 
+    runs-on: ubuntu-18.04
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'python', 'java']
+        # CodeQL supports [ 'cpp', 'csharp', 'go', 'java', 'javascript', 'python' ]
+        # Learn more...
+        # https://docs.github.com/en/github/finding-security-vulnerabilities-and-errors-in-your-code/configuring-code-scanning#overriding-automatic-language-detection
+
+    steps:
+    - name: Checkout repository
```

Comment:
spacings are wrong

Reply:
Spacings have been changed to be consistent with other workflows.